mm2src/mm2_db/src/indexed_db/indexed_db.rs

```diff
@@ -463,6 +463,7 @@ impl<Table: TableSignature> DbTable<'_, Table> {
         items
             .into_iter()
             .map(|(item_id, item)| {
+                web_sys::console::log_1(&format!("Item {}", json::to_string(&item).unwrap()).into());
```

Comment:
Do we still need this printing? How about replacing it with `common::log::debug` then?

Reply:
Oh, I forgot to remove this debug print, it's not required :)